jetty-alpn/jetty-alpn-conscrypt-server/pom.xml

```diff
@@ -38,6 +38,32 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+
```

Comment:
No need for this empty line - all the `test` dependencies should stick together.